### DISCUSSION TITLE
Monitor battery devices

### DISCUSSION THREAD

**Discussion by abertram** [JoDehli/PyLoxone#345] [General]
Hi,

I own a few battery devices like Loxone presence sensor or Xiaomi temperature sensors which are connected to Loxone. My Loxone system is connected to Home Assistant via PyLoxone. I would like to monitor the battery devices in Home Assistant by integrations like Battery Notes to be notified to change the battery. It doesn't work as expected as Battery Notes doesn't recognize the battery devices. I assume becauce Battery Notes scans for entities of type battery and doesn't find anything.

How could I solve the issue?

Best,
Alex

**Comment by JoDehli** [comment]
@abertram What devices are the xiaomi sensors in loxone

**Comment by abertram** [comment]
The devices are temperature and humidity sensors which were originally bluetooth devices. I converted them to Zigbee devices and connected them via LoxBerry and Zigbee2MQTT to Loxone.

**Reply by JoDehli** [reply]
@abertram if you have them already in mqtt then read it from home assistant direct

**Reply by abertram** [reply]
I already have the devices integrated in Home Assistant via PyLoxone. But I would like to use already existing integrations like Battery Notes or other blueprints if possible.
